mmv1/templates/terraform/constants/cloud_dataproc_batch.go.erb

```diff
@@ -0,0 +1,15 @@
+/*
+ * Dataproc Batch api apends subminor version to the provided
+ * version. We are suppressing this server generated subminor.
+ */
+func cloudDataprocBatchFieldsDiffSuppressFunc(old, new string) bool {
```

Comment:
Please add some [unit tests](https://googlecloudplatform.github.io/magic-modules/develop/test/test/#add-unit-tests) for this function to make sure it's doing what you expect in all cases